vision_agent/tools/tools.py

```diff
@@ -2625,6 +2626,187 @@ def _plot_counting(
     return image
 
 
+class ODModels(str, Enum):
```

Comment:
Should we add in florence2 here as well?

Reply:
Done